governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+- MUST adhere to the [Technical Leadership Principles].  
+- SHOULD remain active and responsive in their Roles.
+
+### Formation
+
+Subprojects require a charter describing their purpose, scope, roles, and any regular deliverables
+or services they provide.  This charter MUST be approved by the TOC. 
+
+It is strongly recommended to share a draft charter and work with the TOC early to ensure a
+streamlined approval process.
+
+## Initiatives
+
+Initiatives are short-term, time bound work that have a pre-defined objective and exit criteria.
+Some examples include white papers, one-off reports, related to a subproject, or working with the
+TOC on a specific initiative (e.g. DTR). They do not have a charter, or defined leads, but can have
+separate meetings if it is beneficial to the initiative completing its goal. Each initiative MUST
+have a Tech Lead assigned to oversee and ensure timely completion and closure of the work within
+the scope of the TAGs charter.
+
+### Goals
+
+- Serve as a lightweight organizational unit to coordinate work.  
+- Recognize community members who have helped execute and deliver on the initiative.  
+- Ensure high-velocity completion of scoped effort with clear outcomes aligned with the CNCF mission
+  and TOC vision
+
+### Requirements
+
+- MUST be documented, including goals, milestones, deliverables, and any exit criteria (e.g., a
+  tracking GitHub issue).  
+- Initiatives MUST have more than two participants not including the assigned lead for oversight.  
+- SHOULD actively be worked on. An initiative that has made little progress in 3 months should be
+  canceled or paused until other contributors can be identified.   
+- Initiatives SHOULD not last longer than 2 quarters but MAY request an extension from their parent
+  governing body with reasonable justification.  
+- MUST check in at least monthly to assess the initiative's continued need, progress, and ability
+  to meet.  
+- SHOULD acknowledge everyone who actively helped complete the initiative.  
+- Subproject based initiatives MAY have different requirements that are outlined in the subproject
+  charter.
+
+### Roles
+
+ Assigned Tech Leads, Subproject Leads or TOC members serve as the general oversight of initiatives
+ depending on the parent governing body.
+
+### Formation
+
+There are three types of initiatives with different formation requirements, TAG initiatives,
+subproject initiatives, and TOC initiatives.
+
+#### TAG Initiatives
+
+General initiatives can be any "piece of work," such as white papers, reports, etc., that align
+with the scope of a TAG. The Tech Leads manage TAG Initiatives and MUST have at least one assigned
+as the sponsor to oversee the initiative's execution. They MAY delegate some of the responsibility
+of organizing and shepherding the initiative, but are still ultimately responsible for it.
+
+TAG initiatives can span multiple TAGs when something spans multiple tech domains, however one is
```

Comment:
This section says it's okay to `span multiple TAGs` but then says that it shouldn't `span many TAGs`, which reduces clarity. If possible, a specific threshold would help. Otherwise, consider just rephrasing to shift some of the ambiguity away.

```suggestion
TAG initiatives can invite contribution from multiple TAGs when something spans multiple tech domains, however one is
```